sda-commons-server-kafka/src/main/java/org/sdase/commons/server/kafka/KafkaProperties.java

```diff
@@ -18,6 +21,33 @@ private KafkaProperties() {
     //
   }
 
+  private static Properties configureSecurity(Security security) {
+    KafkaProperties props = new KafkaProperties();
+
+    if (security == null) {
+      return props;
+    }
+
+    if (security.getProtocol() != null) {
+      props.put("security.protocol", security.getProtocol().name());
+    }
+
+    if (security.getPassword() != null && security.getUser() != null) {
+      props.put("sasl.mechanism", security.getSaslMechanism());
+      props.put(
+          "sasl.jaas.config",
+          getLoginModule(security.getSaslMechanism())
+              .getName()
+              .concat(" required username='")
+              .concat(security.getUser())
+              .concat("' password='")
+              .concat(security.getPassword())
+              .concat("';"));
```

Comment:
NIT: Although this is the same as before, I think this is one of the rare cases where even I think, `String.format` is the most readable option. Personally I would prefer `String.format` over the `+` operator over `concat` in this specific case.

Reply:
Yes. `String.format` would be perfect here.